### PR TITLE
[Translation Support] Apply translations (screens/Home)

### PR DESCRIPTION
**Pull request by micahtyong**
# In this PR

In this PR, we follow the steps in the ["Applying Our Hook" section](https://www.notion.so/calblueprint/Translation-Support-Design-Document-7703018a8c4e403dab8aaa49d2a322e7#a7e5cbd6962e4dbd850b724c3eee7b18) of our [roadmap for translation support](https://www.notion.so/calblueprint/Translation-Support-Design-Document-7703018a8c4e403dab8aaa49d2a322e7#cfc52fc0865e4bb2b140e54d395343ba) in order to translate `screens/Home` for `meepanyar`. 

## Coverage
- [ ] `screens/Customers`
- [ ] `screens/Camera`
- [ ] `screens/FinancialSummary`
- [x] `screens/Home`
- [ ] `screens/Inventory`
- [ ] `screens/Profile`

## How to Review
1. Change your browser device setting to Burmese (or return "my" in `device.ts#detectLanguage`)
2. Run the application and navigate to the screens corresponding to `screens/Home`. 
3. Ensure nothing is breaking
4. Test in English and ensure wording is still correct

We will translate the rest of `screens/*` in subsequent PRs. 

cc: @julianrkung

### CODE REVIEW

**Review by julianrkung — COMMENTED** [review]
Hey! Home was just updated (late PR got in). Would you mind adjusting it for the new home screen? Sorry I should've warned that it was in progress.

File: src/screens/Home/Home.tsx [hunk 1]
```diff
@@ -102,16 +105,16 @@ function Home(props: HomeProps) {
           <div style={{ flex: 2 }}>
             <SiteMenu currentSite={currentSite} sites={allSites} />
             <Typography color='secondary'>
-              Current Period: {formatDateStringToLocal(getCurrentPeriod(), true)}
+              {intl(words.current_x, words.period)}: {formatDateStringToLocal(getCurrentPeriod(), true)}
             </Typography>
             <Typography color='secondary'>
-              Deadline: {formatDateStringToLocal(getNextPeriod(), true)}
+              {intl(words.next_deadline)}: {formatDateStringToLocal(getNextPeriod(), true)}
```
Comment (micahtyong) [inline src/screens/Home/Home.tsx:111]: I'm trying to be a bit more conservative when it comes to adding new translation labels due to our recent email exchanges with Sierra. Let me know if this (or any of my labels) strongly conflicts with the current Figma design. cc: @julianrkung

**Comment by micahtyong** [comment]
Before testing, simply run `npm run translation-sync` to get rid of merge conflicts for `resources.js` and `words.ts`.

File: src/lib/i18next/resources.js [hunk 2]
```diff
@@ -265,7 +272,7 @@ module.exports = {
       "Reopen": "ပြန်ဖွင့်မည်",
       "Report": "အစီရင်ခံစာ",
       "Reported By": "တင်သွင်းသူ",
-      "Reports": "အစီရင်ခံစာများ",
+      "Reports": "Reports",
```
Comment (julianrkung) [inline src/lib/i18next/resources.js]: Why did this get overridden? If there are translations existing we shouldn't override right?

File: src/screens/Home/Home.tsx [hunk 3]
```diff
@@ -102,16 +105,16 @@ function Home(props: HomeProps) {
           <div style={{ flex: 2 }}>
             <SiteMenu currentSite={currentSite} sites={allSites} />
             <Typography color='secondary'>
-              Current Period: {formatDateStringToLocal(getCurrentPeriod(), true)}
```
Comment (julianrkung) [inline src/screens/Home/Home.tsx:105]: Line 75 could you also translated the `Owed to MP` and `Ks`?

**Review by julianrkung — COMMENTED** [review]
Looks good :). Had a question about the `resources` that was generated and then also a missed translation but we can merge right after those are fixed!

**Comment by julianrkung** [comment]
Also, I was getting this error when I checked out your branch. Is this just cause it's not updated with main?

image

File: src/lib/i18next/resources.js (hunk 2 above)
Comment (micahtyong) [reply src/lib/i18next/resources.js]: Resolved!

**Review by julianrkung — APPROVED** [review]
Great work :))). It's really cool seeing the translations!

image

**Pull request merged**